app/src/main/java/ch/epfl/sweng/eventmanager/ui/event/interaction/EventShowcaseActivity.java

```diff
@@ -48,9 +48,9 @@
     private SpotsModel spotsModel;
     private ZoneModel zonesModel;
     private Fragment eventMainFragment;
-    private Fragment eventMapFragment;
-    private Fragment newsFragment;
-    private Fragment scheduleParentFragment;
+    public Fragment eventMapFragment;
+    public Fragment newsFragment;
+    public Fragment scheduleParentFragment;
```

Comment:
I think this should remain private

Reply:
If they are private, they cannot be used by other activities such as EventMainFragment to recycle the created fragments (if any). What I wanted to do was reusing the already created fragments instead of recreating them every time.